.tool-versions

```diff
@@ -3,11 +3,10 @@
 # Be EXTREMELY CAREFUL with this. If you override a standard version
 # you are reducing compatibility guarantees.
 ## <<Stencil::Block(toolverOverride)>>
-## <</Stencil::Block>>
-golang 1.22.0
+golang 1.23.0
 nodejs 20.12.2
-protoc 21.5
-terraform 1.5.7
+mage latest
```

Comment:
I’m not sure if asdf supports “latest”, vs mise. Also, with devbase mage is installed for you already. Probably just not working because of our env setups

Reply:
doh, k fixed, putting mage with an explicit version, and in after the default options